pkg/operator/config.go

```diff
@@ -228,5 +235,15 @@ func ConfigFromEnvironment() *Config {
 		config.WebhooksCertDir = webhooksCertDir
 	}
 
+	if metricsPort, ok := os.LookupEnv("METRICS_PORT"); ok {
+		v, err := strconv.ParseInt(metricsPort, 10, 32)
+		if err != nil {
+			v = DefaultMetricsPort
+			klog.Errorf("Error parsing METRICS_PORT environment variable: %v", err)
```

Comment:
Log the value that was parsed, but failed:

`klog.Errorf("Error parsing %q METRICS_PORT environment variable: %v", os.LookupEnv("METRICS_PORT"), err)`